.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -48,3 +48,4 @@ Text from here to the end of the body will be considered for inclusion in the re
 - [ ] Did you review?
 - [ ] Did you check whether all relevant Contributor checkboxes have been checked?
 - [ ] Did you mark as `Please Merge`?
+- [ ] Did you `Enable auto-merge (squash)` and clean up the commit message? (ignore if merging with merge commit)
```

Comment:
can we put this before the "did you mark as `Please Merge`"? I feel like that should be the last one...

I forget how our merge rules work. will GitHub's auto-merge NOT merge unless you've labeled with `Please Merge`? If not, we should probably just get rid of the `Please Merge` label and automation...

Reply:
Github's auto-merge will merge even without `Please Merge` but only if the PR is up-to-date. We still need Mergify to merge master to get PRs up-to-date.

Reply:
so does `Please Merge` automatically merge master?

Reply:
basically i think this checklist is a bit confusing, maybe we should add a header "WHEN YOU REALLY WANT TO MERGE IT" then those last two checklist boxes, in swapped order?

Reply:
(you already swapped the order, the order is good now...)

Reply:
i made a suggest change.... it seems like we have 2 divergent paths for Merge vs Squash commit now.

Reply:
Technically they've always been divergent but not clearly captured there because if you're doing the "Rebase" mode, the contributor is supposed to rebase to clean up the history and update it to master (no `Please Merge`), but yeah this should be made more clear